packages-ts/starknet-gauntlet-cli/networks/.env.goerli

```diff
@@ -0,0 +1 @@
+NODE_URL=https://goerli.infura.io/v3/7c43471f9d604276a856f0cff1edb645
```

Comment:
I assume committing this file was not intentional?

Reply:
I replaced the the infura url with the public one. 
For convenience purposes we normally commit network specific `.env` files (there are few in this folder) with publicly available endpoints. The corresponding env is used when specifying `--network` flag.